packages/client/src/pages/LeaderBoard/LeaderBoard.tsx

```diff
@@ -1,7 +1,71 @@
 import React from 'react'
+import { Box, Typography } from '@mui/material'
+import { leadersMock } from './mock'
+import back from '../../assets/chessboard-background.png'
+import './leaderboard.scss'
 
-const LeaderBoard = () => {
-  return <div>LeaderBoard</div>
+interface IBlockLeader {
+  avatar: string
+  username: string
+  displayName: string
+  score: number | undefined
+  place: number
 }
 
+const LeaderBox: React.FC<IBlockLeader> = (props: IBlockLeader) => {
+  const { avatar, displayName, username, score } = props
```

Comment:
_Не обязательно исправлять_

Если ты указываешь тип пропсов внутри `React.FC`, то внутри круглых скобок это делать уже не обязательно

Плюс, ты можешь достать пропсы через деструктуризацию сразу в круглых скобках

```suggestion
const LeaderBox: React.FC<IBlockLeader> = ({ avatar, displayName, username, score }) => {
```

Reply:
исправил